### PR TITLE
update JSONSchema to Draft v7

### LINKED ISSUE #1808: Migrate to JSONschema Draft v7
Currently we use Draft4. The library that we use supports Draft7

https://json-schema.org/specification.html

The idea would be:
* to use `Draft7Validator` instead of `Draft4Validator` (nothing more...)
* list main changes and add links in CHANGELOG (see spec release notes)
* check compatibility with kinto-admin

### PR DESCRIPTION
**Pull request by peterr101**
Fixes #1808

- [ ] Add documentation.
- [ ] Add tests.
- [x] Add a changelog entry.
- [x] Add your name in the contributors file.
- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)
- [ ] If you added a new configuration setting, update the [`kinto.tpl`](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl) file with it.

### CODE REVIEW

**Comment by leplatrem** [comment]
Draft7 is available is jsonschema v3, you should upgrade it ;) https://github.com/Julian/jsonschema/blob/master/CHANGELOG.rst

**Comment by gabisurita** [comment]
Hey, looks like the tests are breaking because jsonschema>=3 is yet not supported by `bravado_core`. I've opened an issue for this matter.

https://github.com/Yelp/bravado-core/issues/303

**Comment by peterr101** [comment]
Ok, so I am blocked on this issue or is there something else you would like me to try in the meantime?

**Comment by gabisurita** [comment]
@peterr101 The issue was already fixed. We probably just need to wait for them to bump a release.

https://github.com/Yelp/bravado-core/pull/304

Meanwhile, testing with the current `bravado_core` master may be a good idea just to make sure that everything works well.

**Comment by leplatrem** [comment]
@peterr101 some new versions of bravado core were released... could you please have a look?

**Comment by peterr101** [comment]
I updated the bravado_core to the most recent version that has the appropriate JSONSchema compatibility. The changelog and contributors list have also been updated; none of the other items on the checklist seem relevant? Is there anything else I may have missed before this commit can be merged?

File: requirements.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ idna==2.7
 iso8601==0.1.12
 jsonpatch==1.23
 jsonpointer==2.0
-jsonschema==2.6.0
+jsonschema==3.0.0a3
```
[the remark this answers is not in the document]
Comment (peterr101) [reply requirements.txt]: For future knowledge how could I have known that this line would have been needed to be added as a dependency to this file as well? What steps could I have taken to figure this out on my own (new to open source and would like to learn more)?

[the remark this answers is not in the document]
Comment (leplatrem) [reply requirements.txt]: Unfortunately I don't have a strong tip to give you... I looked at the failing test and saw that it was relying on a specific list of deps...

File: CHANGELOG.rst [hunk 2]
```diff
@@ -37,12 +37,11 @@ This document describes changes between each past release.
 - Fix missing favicon and inline images in kinto-admin plugin
 
 **Internal changes**
-
+- Migrate JSON Hyper-Schema to Draft-07 (#1808)
```
Comment (leplatrem) [inline CHANGELOG.rst]: This should go in `11.2.0 (unreleased)`

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterr101) [reply requirements.txt]: I see - ok no worries!

**Pull request merged**